README.md

```diff
@@ -1,13 +1,13 @@
+This repo contains Dan Le's portfolio and STEP Projects.
 # Google Student Training in Engineering Program
-
 This repo contains the projects you'll work on for the first half of STEP.
 This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
-
 To get started:
-
 - Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
 - Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
 - Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
   - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
 Then follow the on-screen instructions to set up your repo.
+dhle@cloudshell:~/step$ nano README.md
```

Comment:
I think this might be an accidental change, please remove. If they're purposeful instructions they could be a little more clear.